packages/sirius-web/frontend/sirius-web/src/nodes/EllipseNodeLayoutHandler.ts

```diff
@@ -50,6 +51,8 @@ export class EllipseNodeLayoutHandler implements INodeLayoutHandler<NodeData> {
     const nodeElement = document.getElementById(`${node.id}-ellipseNode-${nodeIndex}`)?.children[0];
     const borderWidth = nodeElement ? parseFloat(window.getComputedStyle(nodeElement).borderWidth) : 0;
     const labelElement = document.getElementById(`${node.id}-label-${nodeIndex}`);
+    const withHeader: boolean = node.data.label?.isHeader ?? false;
+    const displayHeaderSeparator: boolean = node.data.label?.displayHeaderSeparator ?? false;
```

Comment:
Not sure these make sense for an ellipse, especially `displayHeaderSeparator`.

Reply:
Done